compareECLFiles.cmake

```diff
@@ -54,6 +69,8 @@ endmacro (add_test_compareECLRestartFiles)
 # Input:
 #   - casename: basename (no extension)
 #
+# Details:
+#   - This test class compares the init file from a simulation to a reference file.
```

Comment:
The names of these macros, and the types of tests they set up are slightly misleading; that in partiuclar applies to `add_test_compareECLRestartFiles` - most of the tests compare restart files. The purpose of this macro is to create tests for testing restarted simulations; I would suggest e.g. `add_test_compare_restarted_simulation( )`.